### ISSUE TITLE
Wrong schema version in config

### ISSUE THREAD

**Issue by ryssbowh** [la-haute-societe/craft-restrict-asset-delete#5]
I'm using this plugin on many websites, it's the second time now that I see this happening : 
- Update to 1.1.4 (I was in 1.1.2)
- Apply config in another environment through command line, this is the result :
`Your project config files were created for different versions of Craft and/or plugins than what’s currently installed.
Restrict Asset Delete is installed with schema version of 1.1.0 while 1.0.0 was expected.`

In v1.1.3 was introduced to src/RestrictAssetDelete.php the line 38 : `public $schemaVersion = '1.1.0';`
The default would be 1.0.0, so that module changed schema version but is not introducing new migrations. I think that line should be deleted.

I haven't tried but I think that's what's wrong.

The workaround in the meantime is to set the schema version manually to 1.1.0 in config/project/project.yaml

Thanks

**Comment by nstcactus** [comment]
The 1.1.0 schema version is "intended". I took over this repo and inherited its code. A few weeks ago, [I accidentally removed the `$schemaVersion = '1.1.0';` property of the plugin](https://github.com/la-haute-societe/craft-restrict-asset-delete/commit/48b7affb99f725053e09163a7e5e07e62a2ef0af#diff-216389f7e68e3fef0b6142b13eb82a04e916f53138d56932503f7943f2928e34L63). The fact this plugin doesn't use database storage made me think it was set to the default `1.0.0` value and that it could be safely removed.

This caused a bug for users of pre 1.1.2 versions similar to yours:
> Your project config files were created for different versions of Craft and/or plugins than what’s currently installed. Restrict Asset Delete is installed with schema version of 1.0.0 while 1.1.0 was expected.

This is why I had to [re-add the `$schemaVersion = '1.1.0';` property](https://github.com/la-haute-societe/craft-restrict-asset-delete/commit/24f0de6f330b8a1c28b9c780bbdaacdf3202776c).

Setting the correct version in _config/project/project.yaml_ is the right thing to do although I have no idea why Craft didn't do it when you updated the plugin.

Just out of curiosity, which method did you use to update the plugin?
- `craft update all` command
- **Utilities** / **Updates** section of the control panel
- plain `composer update`
- …
